### PR TITLE
[10.0][ADD] account_payment_mode : relational inverse field for payment.mode on payment.method

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

File: account_payment_mode/models/account_payment_method.py [hunk 1]
```diff
@@ -22,6 +22,8 @@ class AccountPaymentMethod(models.Model):
     display_name = fields.Char(
         compute='compute_display_name',
         store=True, string='Display Name')
+    payment_mode_ids = fields.One2many(
+        'account.payment.mode', 'payment_method_id', 'Payment modes')
```
Comment (adrienpeiffer) [inline account_payment_mode/models/account_payment_method.py]: I find it better to name these arguments

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

File: account_payment_mode/models/account_payment_method.py (hunk 1 above)
Comment (adrienpeiffer) [reply account_payment_mode/models/account_payment_method.py]: @grindtildeath Have you seen my comment ?

Comment (grindtildeath) [reply account_payment_mode/models/account_payment_method.py]: @adrienpeiffer Sorry I missed it, will fix it asap

**Comment by adrienpeiffer** [comment]
@grindtildeath Thanks ! Can you squash the last commit ?

**Comment by grindtildeath** [comment]
@adrienpeiffer Done :)

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can you please change module version number?

**Review by benwillig — APPROVED** [review]

**Comment by grindtildeath** [comment]
@pedrobaeza Version number increased, commits squashed.

**Pull request merged**